web_external/Studies/studyPage.pug

```diff
@@ -28,14 +28,25 @@
       td.isic-study-users-content
         if study.get('users').length
           table(class=study.canAdmin() ? 'table-hover' : undefined)
+            th(colspan=2)
+            th Num. Completed
+            th Percent Completed
             each user in study.users().models
+              - var numCompleted = 'N/A'
```

Comment:
We can start using `let` instead of `var` here, since templates are passed through Babel.